test/plugin/scenarios/redisson-scenario/bin/startup.sh

```diff
@@ -18,4 +18,4 @@
 
 home="$(cd "$(dirname $0)"; pwd)"
 
-java -Dredis.servers=${REDIS_SERVERS} -Dskywalking.plugin.redisson.trace_redis_parameters=true -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
\ No newline at end of file
+java -Dredis.servers=${REDIS_SERVERS} -Dskywalking.plugin.redisson.trace_redis_parameters=true -Dskywalking.plugin.redisson.show_batch_commands=true -Dskywalking.plugin.redisson.ping_ignored=true -jar ${agent_opts} ${home}/../libs/redisson-scenario.jar &
```

Comment:
`skywalking.plugin.redisson.ping_ignored=true`. I think this is the default value, why set it up again?

Reply:
I'm going to remove duplicate actions.